performance_regression_test.py

```diff
@@ -99,7 +105,35 @@ def get_test_xml(self, result):
 
         return test_content
 
-    def display_results(self, results):
+    def generate_stats_json(self, results, cmds=[]):
+        metrics = {}
+
+        for p in self.params.iteritems():
+            metrics[p[1]] = p[2]
+
+        # we use cmds
+        del metrics['stress_cmd']
+        for cmd in cmds:
+            metrics = self.add_stress_cmd_params(metrics, cmd)
+            metrics = self.add_stress_cmd_params(metrics, cmd)
+
+        metrics['test_name'] = self.params.id.name
+        metrics['stats'] = results
+
+        if 'es_password' in metrics:
+            # hide ES password
+            metrics['es_password'] = '******'
```

Comment:
from PR descr:
```to upload test stats in ES we need specify the following parmas in yaml file:
es_url: https://HOST_NAME:9243
es_user: elastic
es_password: password
```
so, account info is stored in yaml config. I use " if 'es_password' in metrics" here because I've already set all yaml's params in metrics variable.